src/layer/LayersFactory.ts

```diff
@@ -46,14 +48,6 @@ export class LayersFactory {
     instantiate appropriate layers.
   */
 
-  public static readonly SH_SERVICE_HOSTNAMES_V1_OR_V2: string[] = ['https://eocloud.sentinel-hub.com/'];
-
-  public static readonly SH_SERVICE_HOSTNAMES_V3: string[] = [
-    'https://services.sentinel-hub.com/',
-    'https://services-uswest2.sentinel-hub.com/',
-    'https://creodias.sentinel-hub.com/',
-  ];
-
   private static readonly DATASET_FROM_JSON_GETCAPAPABILITIES = {
```

Comment:
1. This should be named `DATASET_FROM_JSON_GETCAPAPABILITIES_V3`

2. Should this and `DATASET_FROM_JSON_GETCAPABILITIES_V1`, `LAYER_FROM_DATASET_V3`, `LAYER_FROM_DATASET_V12` also be moved to `./const.ts`?

Reply:
1. Agree, not critical though

2. Not sure, tbh... Maybe we can leave it here for now? If it bothers we can always move it later.

Reply:
Ok, I will leave that for later. Resolving